tests/Dockerfile

```diff
@@ -1,35 +1,42 @@
 FROM registry.access.redhat.com/ubi8/ubi:latest
 
-RUN dnf install -y bc git go-toolset && \
-    git clone https://github.com/crobby/peak && \    
-    cd /peak && \
+ARG ORG=opendatahub-io
+ARG BRANCH=master
+
+RUN dnf install -y bc git go-toolset
+
+ENV HOME /root
+WORKDIR /root
+
+RUN git clone https://github.com/crobby/peak && \
+    cd peak && \
     git submodule update --init
 
-RUN mkdir -p /src && \
-    cd /src && \
-    git clone https://github.com/opendatahub-io/odh-manifests && \
-    chmod -R 777 /src
-
-ADD https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/linux/oc.tar.gz /peak/ 
-RUN tar -C /usr/bin -xvf /peak/oc.tar.gz && \
-    chmod +x /usr/bin/oc
-
-COPY basictests /peak/operator-tests/odh-manifests/basictests
-COPY functionalitytests /peak/operator-tests/odh-manifests/functionalitytests
-COPY util /peak/operator-tests/odh-manifests
-COPY resources/kfctl_openshift.yaml /kfdef/
-COPY resources/operatorsetup /peak/
-COPY resources/ai_library_cr.yaml /peak/operator-tests/odh-manifests/resources/
-COPY scripts/install.sh /peak/
-COPY scripts/installandtest.sh /peak/
-
-RUN chmod -R 777 /kfdef && \
-    mkdir -p /.kube && \
-    chmod -R 777 /.kube && \
-    chmod -R 777 /peak
+RUN mkdir -p src && \
+    cd src && \
+    git clone --depth=1 --branch ${BRANCH} https://github.com/${ORG}/odh-manifests && \
+    chmod -R 777 $HOME/src
+
+ADD https://mirror.openshift.com/pub/openshift-v4/clients/oc/latest/linux/oc.tar.gz peak/
+RUN tar -C /usr/local/bin -xvf peak/oc.tar.gz && \
+    chmod +x /usr/local/bin/oc
+
+COPY basictests peak/operator-tests/odh-manifests/basictests
+COPY functionalitytests peak/operator-tests/odh-manifests/functionalitytests
+COPY util peak/operator-tests/odh-manifests
+COPY resources/kfctl_openshift.yaml kfdef/
+COPY resources/operatorsetup peak/
+COPY resources/ai_library_cr.yaml peak/operator-tests/odh-manifests/resources/
+COPY scripts/install.sh peak/
+COPY scripts/installandtest.sh peak/
+
+RUN chmod -R 777 kfdef && \
+    mkdir -p .kube && \
+    chmod -R 777 .kube && \
+    chmod -R 777 peak
 
 # For local testing, you can add your own kubeconfig to the image
 # Note:  Do not push the image to a public repo with your kubeconfig
 # ADD kubeconfig /root/.kube/config
 
-CMD /peak/installandtest.sh
+CMD $HOME/peak/installandtest.sh
```

Comment:
Since the ci-tests seem to ignore/override this anyway, maybe we should default to something like `$HOME/peak/run.sh basictests` so the default behavior when running locally will be to just run the test suite?